winconf.pri

```diff
@@ -53,7 +53,7 @@ CONFIG(debug, debug|release) {
 # Enable backtrace support
 CONFIG += strace_win
 
-win32-g++ {
+*-g++* {
```

Comment:
The original string is `win32-g++` and you change it to `*-g++*`. I am talking about changing it to `win32-g++*` which means any g++ on Windows.
The question is will `win32-g++*` work for MXE? Can you test it? If it does work then change it to what I suggested.